mandible/log.py

```diff
@@ -1,9 +1,68 @@
+import json
 import logging
 import os
 from contextlib import contextmanager
+from functools import wraps
 from typing import Type
 
 
+class JSONFormatter(logging.Formatter):
+    def format(self, record):
+        log_record = {
+            "aws_request_id": getattr(record, "aws_request_id", None),
+            "cirrus_core_version": getattr(record, "cirrus_core_version", None),
+            "cirrus_daac_version": getattr(record, "cirrus_daac_version", None),
+            "cumulus_version": getattr(record, "cumulus_version", None),
+            "function_name": getattr(record, "function_name", None),
+            "granule_name": getattr(record, "granule_name", None),
+            "invoked_function_arn": getattr(record, "invoked_function_arn", None),
+            "level": record.levelname,
+            "log_group_name": getattr(record, "log_group_name", None),
+            "log_stream_name": getattr(record, "log_stream_name", None),
+            "memory_limit_in_mb": getattr(record, "memory_limit_in_mb", None),
+            "message": record.getMessage(),
+            "step_function_name": getattr(record, "step_function_name", None),
+            "time": self.formatTime(record, self.datefmt),
```

Comment:
Looking at the example payload I think these are extraneous:

```
function_name -- already present in log group name
invoked_function_arn -- you can get the arn from the name
log_group_name
log_stream_name
```

I think `step_function_name` would be better as `execution_name` or `workflow_execution`. Something with the word `execution` in it as that's what it's called in cumulus. The step function name would be the workflow name e.g. `OPERAIngestWorkflow`.